src/main/scala/it/bitrock/dvs/streams/topologies/FlightInterpolatedListStream.scala

```diff
@@ -0,0 +1,101 @@
+package it.bitrock.dvs.streams.topologies
+
+import java.util.Properties
+
+import it.bitrock.dvs.model.avro.{FlightReceived, FlightReceivedList}
+import it.bitrock.dvs.streams.StreamProps.streamProperties
+import it.bitrock.dvs.streams._
+import it.bitrock.dvs.streams.config.AppConfig
+import it.bitrock.dvs.streams.geo.utils.EarthPositionCalculator
+import org.apache.kafka.common.serialization.Serde
+import org.apache.kafka.streams.kstream.Transformer
+import org.apache.kafka.streams.processor._
+import org.apache.kafka.streams.scala.ImplicitConversions._
+import org.apache.kafka.streams.scala.StreamsBuilder
+import org.apache.kafka.streams.state.{KeyValueStore, Stores}
+import org.apache.kafka.streams.{KeyValue, Topology}
+
+import scala.concurrent.duration._
+
+object FlightInterpolatedListStream {
+  private val currentSnapshot = "currentSnapshot"
+
+  def buildTopology(config: AppConfig, kafkaStreamsOptions: KafkaStreamsOptions): List[(Topology, Properties)] = {
+    implicit val stringKeySerde: Serde[String]                           = kafkaStreamsOptions.stringKeySerde
+    implicit val flightReceivedListEventSerde: Serde[FlightReceivedList] = kafkaStreamsOptions.flightReceivedListEventSerde
+
+    val streamsBuilder = new StreamsBuilder
+
+    val stateStore = Stores.keyValueStoreBuilder(
+      Stores.persistentKeyValueStore("FlightListInterpolationStateStore"),
```

Comment:
I'd extract the name in a constant

Reply:
Yep, I was thinking about the same thing even if it's used only one time